### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by clseibold** [wolfSSL/go-wolfssl#8]
It seems like the cgo directives in the code assume Linux. Is there a way to get this working on Windows?

**Comment by kareem-wolfssl** [comment]
Hi @clseibold,

We expect this to work on Windows if you use something like WSL or Mingw64, but it has not been tested yet.  Please let us know if you run into any issues.

**Comment by clseibold** [comment]
Yeah, not using WSL or mingw64, sorry. WolfSSL can compile in VS 2008, so I expect the golang *wrapers* to be able to wrap around an already-compiled library.

**Comment by kareem-wolfssl** [comment]
I've logged a feature request for this in our internal tracker.  We implement feature requests on a best-effort basis.  If you'd like to expedite this request please contact us at support@wolfssl.com.
If you do end up forking you are welcome to contribute your changes.
